### PR TITLE
Add 2nd result list which shows answers from neural.api.netspeak.org

### PR DESCRIPTION
**Pull request by MattiWe**

### CODE REVIEW

File: src/page-elements/search-page.tsx [hunk 1]
```diff
@@ -118,32 +126,69 @@ export default class SearchPage extends React.PureComponent<unknown, State> {
 		});
 	};
 
+	private _onShowExperimental = (): void => {
+		this.setState({ betaResults: !this.state.betaResults });
```
Comment (RunDevelopment) [inline src/page-elements/search-page.tsx:130]: This isn't guaranteed to work. Use an updater function instead.
```suggestion
		this.setState(state => ({ betaResults: !state.betaResults }));
```

File: src/page-elements/search-page.tsx [hunk 2]
```diff
@@ -29,6 +30,9 @@ interface State {
 	corpora: readonly Corpus[];
 	unavailableCorpora: ReadonlySet<Corpus>;
 
+	betaResults: boolean;
+	refreshSearch: boolean; //set to true after pressing the showExperimental button to refresh the result list results
```
Comment (RunDevelopment) [inline src/page-elements/search-page.tsx:34]: This is unnecessary. The value is never changed, so you can remove this from the state.

File: src/elements/netspeak-search.scss [hunk 3]
```diff
@@ -1,14 +1,25 @@
 .NetspeakSearch {
 	font-size: 100%;
-	border-left: 1px solid #bbb;
-	border-right: 1px solid #bbb;
 
 	& > .wrapper {
 		border-bottom: 1px solid #bbb;
+		border-left: 1px solid #bbb;
+		border-right: 1px solid #bbb;
```
Comment (RunDevelopment) [inline src/elements/netspeak-search.scss:7]: Why?

File: src/elements/netspeak-search.tsx [hunk 4]
```diff
@@ -419,34 +440,41 @@ export class NetspeakSearch extends React.PureComponent<Props, State> {
...
-									<TransparentButton image={url(ClearImage)} onClick={this._onClearButtonClick} />
-								</td>
-								{optional(!!this.props.history, () => (
-									<td>{this._renderHistoryPopup()}</td>
-								))}
-							</tr>
-						</tbody>
-					</table>
-				</div>
+				{optional(this.props.apiType === NetspeakApi.neural, () => (
```
Comment (RunDevelopment) [inline src/elements/netspeak-search.tsx:443]: Controlling whether the search bar is interactable or not via the API kind is really hacky.

File: src/elements/netspeak-search.tsx [hunk 5]
```diff
@@ -102,10 +107,18 @@ export class NetspeakSearch extends React.PureComponent<Props, State> {
...
 		this._queryPhrases(this.state.normalizedQuery);
 	}
+
 	componentWillUnmount(): void {
 		this.cancelable.cancel();
 	}
 
+	componentDidUpdate(prevProps: Props, prevState: State): void {
+		if (prevProps.storedQuery !== this.props.storedQuery) {
+			console.log("update to query: ", this.props.storedQuery);
```
Comment (RunDevelopment) [inline src/elements/netspeak-search.tsx:117]: We don't need to log that in prod :)

File: src/elements/netspeak-search.tsx [hunk 6]
```diff
@@ -1,28 +1,29 @@
...
-	normalizeQuery,
 	Netspeak,
-	NetspeakInvalidQueryError,
+	NetspeakApi,
 	NetspeakError,
+	NetspeakInvalidQueryError,
+	normalizeQuery,
+	Phrase,
 	ReadonlyNetspeakSearchResult,
 } from "../lib/netspeak";
```
Comment (RunDevelopment) [inline src/elements/netspeak-search.tsx:11]: It's a good idea to sort these, but feature and stylistic changes should be separate PRs.

Let's include the stylistic changes for now. I'll make a PR for string imports after merging this one.

**Comment by MattiWe** [comment]
I agree on the mobile design. 
We could either
(1) Hide it, or
(2) Break the layout at the sections. So on <1000px, the Beta-button and the Beta-and the results list appear below the ngram result list. 

This will become harder once the graph is implemented back in.

File: src/elements/netspeak-search.tsx (hunk 6 above)
Comment (RunDevelopment) [reply src/elements/netspeak-search.tsx]: Never heard of that being used in webdev but it works I guess?

**Comment by RunDevelopment** [comment]
Agreed. Can you implement that? I'll try to track down the cause of the CI error in the meantime.

**Pull request merged**